src/sidebars/favourites/tearout-directive.js

```diff
@@ -157,12 +157,11 @@
                         }
 
                         function boundsChangingEvent() {
-                            // Check if you are over a drop target by seeing if the tearout rectangle intersects the drop target
-                            insideFavouritesPane = geometryService.elementIntersect(
-                                tearoutWindow, window, document.getElementsByClassName('favourites')[0]);
-
-                            if (insideFavouritesPane) {
+                            if (dragService.overThisInstance(TEAR_IN_SELECTOR)) {
                                 reorderFavourites();
+                            } else {
+                                // Check intersections to set the tear in indicator states.
+                                dragService.overAnotherInstance(TEAR_IN_SELECTOR);
```

Comment:
I think I've got a solution to this, to hopefully make it all contained in the drag service. I'll update in due course